dnscrypt-proxy/example-allowed-names.txt

```diff
@@ -1,16 +1,16 @@
 
 ###########################
-#        Whitelist        #
+#        Allowlist        #
 ###########################
 
-## Rules for name-based query whitelisting, one per line
+## Rules for name-based query allowlisting, one per line
```

Comment:
Rules for allowing queries based on name?

Reply:
Or i guess keep it consistent with the blocked-names comment.

Reply:
Rules for name-based query allow lists, one per line ?

Reply:
Confusing because it reads as if the allowlist contain rules for allow lists. The file *is* the list.

Reply:
it does -- on it....

Reply:
used your original "allowing queries based on name"